examples/minimal/README.md

```
@@ -0,0 +1,21 @@
+# Minimal
+
+**TODO: Add description**
```

Markdownlint (0.37.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)